### DISCUSSION TITLE
Any ways to next build without having useless files in .next folder?

### DISCUSSION THREAD

**Discussion by chris-lee-code** [vercel/next.js#21796] [Help] [answered]
Hi. I always encounter a problem when building an app using npm run build or next build. Really, I don't want the useless component files in the .next folder. Are there any ways to build an app to deploy without creating such files?

**Comment by jamesmosier** [comment, marked as answer]
There is currently no way to customize the output of `next build`. What are the "useless components" you are seeing? Everything should be needed/used 🤔

**Reply by chris-lee-code** [reply]
I see. Just some npm run build has given me some errors every time. Always the card.js complains that the AdminNavbar is not defined. So I was wondering if I can just build an app without getting those example components javascript files so that I can build the program successfully without any concerns. 

But you are right. We do need some of the files. Just maybe there could be ways later to build an app without any example components but only the build file that is really needed. Thanks for your replies, @jamesmosier, though!👍
